metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1249,6 +1250,24 @@ class MetalsLanguageServer(
             val msg = s"Invalid arguments: $args. Expecting: $argExample"
             Future.failed(new IllegalArgumentException(msg)).asJavaObject
         }
+      case ServerCommands.NewScalaWorksheet() =>
```

Comment:
Would it make sense to move it to a CommandProvider?

Reply:
While body of this method deserves (imho) being refactored to another file/class, it probably worth to be done in another PR